src/main/java/seedu/toluist/controller/FindController.java

```diff
@@ -81,24 +84,30 @@ public CommandResult execute(String command) {
 
     private CommandResult formatDisplay(boolean isSearchByTag, boolean isSearchByName,
                                         String[] keywordList, int foundCount) {
-        String searchParameters;
-        if (isSearchByName && isSearchByTag) {
-            searchParameters = NAME_AND_TAG_MESSAGE;
-        } else if (isSearchByName) {
-            searchParameters = NAME_MESSAGE;
-        } else { //isSearchByTag
-            searchParameters = TAG_MESSAGE;
-        }
+        if (keywordList[0].equals(NULL_PARAMETER)) {
+            return new CommandResult(String.format(LIST_RESULT_MESSAGE_TEMPLATE, foundCount));
+        } else {
```

Comment:
use this idiom
```
`if (a) {
   return stth sth
}
Do sth sth here where a is wrong
```
Notice how `else` is eliminated and u can avoid 1 more indentation level